src/genie/libs/parser/iosxe/show_lisp.py

```diff
@@ -1491,6 +1491,9 @@ class ShowLispServiceMapCacheSchema(MetaParser):
                                             Optional('negative_mapping'):
                                                 {'map_reply_action': str,
                                                 },
+                                            Optional('encap_to_proxy'):
+                                                {'device': str,
+                                                },
```

Comment:
```suggestion
                                            Optional('encap_to_petr'): bool,
                                            Optional('encap_to_petr_iid'): str,
```
adjusting schema as per my comments.